Pod/Classes/PURLogStore.m

```diff
@@ -54,10 +51,15 @@ - (instancetype)initWithDatabasePath:(NSString *)databasePath
     self = [super init];
     if (self) {
         _databasePath = databasePath;
+        _databaseReadWriteCompletionQueue = dispatch_queue_create("PureeLogStoreReadWriteCompletion", NULL);
```

Comment:
Yes.

DB R/W process is too heavy because it also runs disk IO, but in contract, BufferedOutput's internal process is not too heavy. Rather than that, using main-thread makes it much more simpler. 
Anyway, currently `-(void)flush` is being called from multiple thread and may sometimes cause NSRangeException.